STM32/DC/InputValidation/Src/inputValidation.c

```diff
@@ -37,7 +37,7 @@ int getNumberDigits(int inputNumber){
 bool isInputInt(char * inputBuffer, int startIdx, int endIdx) {
 	int idx = startIdx;
 	while (idx <= endIdx) {
-		if (!isdigit(inputBuffer[idx])) {
+		if (!isdigit((int)inputBuffer[idx])) {
```

Comment:
Will this not cause a casting error if it is not a digit? I think it states in the documentation of isdigit() that you have to pass a char.

Reply:
I read the documentation wrong, my bad :)